plugins/modem-manager/fu-sahara-loader.h

```diff
@@ -13,7 +13,7 @@
 G_DECLARE_FINAL_TYPE(FuSaharaLoader, fu_sahara_loader, FU, SAHARA_LOADER, GObject)
 
 FuSaharaLoader *
-fu_sahara_loader_new(void);
+fu_sahara_loader_new(gboolean supports_zlp);
```

Comment:
could you do `void fu_sahara_loader_set_supports_zlp(FuSaharaLoader *self, gboolean supports_zlp);` instead please